pkg/protocol/request.go

```diff
@@ -180,6 +180,14 @@ func (req *Request) ResetSkipHeader() {
 	req.isTLS = false
 }
 
+func (req *Request) ResetSkipHeaderAndBody() {
```

Comment:
"ResetSkipHeader"  & "ResetSkipHeaderAndBody" have the same logic for reset

Reply:
Can't restbody so a new method has been added.😢